src/indexPage/react-components/binary-landing/Hero.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import Slider from 'react-slick';
+import { translate } from '../../../common/i18n';
+
+const carouselImages = [
+    'image/hero-dmt5.png',
+    'image/hero-derivgo.png',
+    'image/hero-derivx.png',
+    'image/hero-dtrader.png',
+]
+
+const carouselSettings = {
+    dots          : false,
+    infinite      : true,
+    speed         : 1000,
+    slidesToShow  : 1,
+    slidesToScroll: 1,
+    arrows        : false,
+    autoplay      : true,
+    autoplaySpeed : 3000,
+    responsive    : [
+        {
+            breakpoint: 1024,
```

Comment:
can we use the breakpoint variables instead of 1024?

Reply:
Thanks @sara-fs, This is actually the carousel setting itself and we are using it once only 🙏